RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -2,7 +2,7 @@
 
 using namespace ALPAKA_ACCELERATOR_NAMESPACE;
 
-constexpr int NUM_HITS = 3;  // Used by T5 DNN
+constexpr int NUM_HITS = 6;  // Used by T5 DNN
```

Comment:
BTW, is this constant even needed? Can you use `Params_T5::kLayers` ?

Reply:
> BTW, is this constant even needed? Can you use `Params_T5::kLayers` ?

I guess not, that would be 5, not 6. 
Why 6, BTW? isn't 2 and 3 the same (IIUC, this would be the actual middle hit)

Reply:
The 6 is just because getHitsFromT3 returns a vector of length 6, but I agree this can be simplified a bit more.

Reply:
since this file is not specific to T5 DNN, please rename `NUM_HITS` to something that's related to T5 hits from T3s

Reply:
I removed NUM_HITS, changed the different arrays to vectors.